pghoard/restore.py

```diff
@@ -294,6 +303,8 @@ def _get_basebackup(self, pgdata, basebackup, site,
             basebackup = self._find_nearest_basebackup(recovery_target_time)
         elif basebackup == "latest":
             basebackup = self._find_nearest_basebackup()
+        elif not isinstance(basebackup, dict):
```

Comment:
isinstance(basebackup, str )?

Reply:
yep - I can do that :), will make the change and push again